web/chromecom.js

```diff
@@ -1,5 +1,4 @@
-/* -*- Mode: Java; tab-width: 2; indent-tabs-mode: nil; c-basic-offset: 2 -*- */
-/* Copyright 2013 Mozilla Foundation
+/* Copyright 2014 Mozilla Foundation
```

Comment:
Can you revert changes of the year of copyright? It's hard to get it right and out of scope of this PR